apps/gustaf/stellar_routines.h

```diff
@@ -282,11 +283,11 @@ void _getMatchDistanceScore(
     typedef typename Iterator<TQueryMatchSet, Standard>::Type TQueryMatchSetIterator;
 
     // TODO(rmaerker): Add threads to options and then remove line below or adapt to parallelization scheme.
-    omp_set_num_threads(1);
+    omp_set_num_threads(numThreads);
     Splitter<TQueryMatchSetIterator> setSplitter(begin(queryMatchesSet, Standard()), end(queryMatchesSet, Standard()));
 
     SEQAN_OMP_PRAGMA(parallel for)
-    for(unsigned jobId = 0; jobId < length(setSplitter); ++jobId)
```

Comment:
This breaks MSVC builds (see http://cdash.seqan.de/viewBuildError.php?buildid=118518). Either use a signed integer or disable gustaf parallelization on MSVC/OpenMP < 3.0.